livekit-agents/livekit/agents/tts/tts.py

```diff
@@ -74,11 +88,11 @@ def synthesize(
         self,
         text: str,
         *,
-        conn_options: APIConnectOptions = DEFAULT_API_CONNECT_OPTIONS,
+        conn_options: Optional[APIConnectOptions] = None,
```

Comment:
DEFAULT_API_CONNECT_OPTIONS is frozen, so it shouldn't be needed to change to Optional

Reply:
yes, but making it required with default forces the pattern of repeating this everywhere.. Plugins should not have to worry about these defaults

it's much cleaner to have a single place that picks the defaults (base class of ChunkedStream).